src/geom_handlers/common_assessment.jl

```diff
@@ -165,7 +165,7 @@ function initial_search_rotation(
     gdf::DataFrame,
     reef_outlines::Vector{Vector{GeometryBasics.Line{2,Float64}}}
 )::Float64
-    distance_indices = filter_far_polygons(gdf, pixel, pixel[2])
+    distance_indices = filter_far_polygons(gdf, pixel, pixel[2], 20000)
```

Comment:
What's the 20000?

Reply:
That's a 20km distance just to filter down so we aren't searching through every reef in the region to find the closest one to a pixel in the next lines. The value is hardcoded at the moment but it's just intended as a buffer to cut down GO.distance run time. 
I can't see it needing to be changed by the user really, unless they were trying to assess a pixel that is > 20km from a reef outline.

Reply:
We call these "magic numbers": values that do not have a meaningful name and/or not documented.

You can imagine myself (or even future Ben) coming back in a few weeks time wondering what `20000` means.

It's better to assign to a variable with a meaningful name.

Reply:
Oh yeah that makes sense, I should have named it clearer. Thanks for fixing